test/run_e2e_test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Please run shellcheck on this script and fix the lint errors, quoting seems to be unsufficient.

Reply:
Done